packages/maskbook/src/plugins/PoolTogether/UI/PoolView.tsx

```diff
@@ -0,0 +1,299 @@
+    },
+    networkIcon: {
+        width: '1em',
+        height: '1em',
+        backgroundColor: 'transparent',
+        marginRight: theme.spacing(0.5),
+    },
+    metaTitle: {
+        marginBottom: theme.spacing(1),
+        justifyContent: 'inherit',
+        alignItems: 'center',
+    },
+    metaFooter: {
+        justifyContent: 'inherit',
+        alignItems: 'center',
+    },
+    metaText: {
+        marginTop: theme.spacing(1),
+        justifyContent: 'inherit',
+    },
+    metaTextPrize: {
+        color: '#55f1d7',
+        margin: theme.spacing(0, 1),
+        backgroundColor: 'rgba(53, 230, 208, 0.2)',
+        borderRadius: theme.spacing(1),
+        padding: theme.spacing(0, 0.5),
+    },
+
+    metaPrize: {
+        marginTop: theme.spacing(1),
+        padding: theme.spacing(1),
+        borderRadius: theme.spacing(1),
+        backgroundColor: '#290B5A',
+        justifyContent: 'center',
+        maxWidth: '50%',
+    },
+    metaDeposit: {
+        marginTop: theme.spacing(1),
+        padding: theme.spacing(0, 1),
+        justifyContent: 'center',
+        maxWidth: '50%',
+    },
+    prize: {
+        background:
+            'linear-gradient(40deg,#ff9304,#ff04ea 10%,#9b4beb 20%,#0e8dd6 30%,#0bc6df 40%,#07d464 50%,#dfd105 60%,#ff04ab 78%,#8933eb 90%,#3b89ff)',
+        '-webkit-background-clip': 'text',
+        color: 'transparent',
+        animation: '$rainbow_animation 6s linear infinite',
+        backgroundSize: '600% 600%',
+        fontSize: '1.2rem',
+        '@media (min-width:600px)': {
+            fontSize: '2rem',
+        },
+    },
+    '@keyframes rainbow_animation': {
+        '0%': {
+            backgroundPosition: '100% 0%',
+        },
+        '100%': {
+            backgroundPosition: '0 100%',
+        },
+    },
+    countdown: {
+        alignSelf: 'center',
+        width: '100%',
+        display: 'flex',
+        justifyContent: 'center',
+    },
+    deposit: {
+        backgroundColor: '#3ef3d4',
+        color: '#4c249f',
+        marginTop: theme.spacing(0.5),
+    },
+    info: {
+        marginTop: theme.spacing(0.5),
+        justifyContent: 'space-between',
+    },
+    apr: {
+        color: '#bdb3d2',
+        display: 'flex',
+    },
+    poolIcon: {
+        backgroundColor: 'transparent',
+        marginRight: theme.spacing(0.5),
+    },
+    viewPool: {
+        cursor: 'pointer',
+        color: '#3ef3d4',
+        textDecoration: 'none',
+        marginRight: theme.spacing(0.5),
+        '&:last-child': {
+            marginRight: 0,
+        },
+        maxHeight: theme.spacing(1),
+        '&:hover': {
+            color: '#ffffff',
+        },
+    },
+}))
+
+interface PoolProps {
+    pool: Pool
+}
+
+export function PoolView(props: PoolProps) {
+    const { pool } = props
+    const classes = useStyles()
+    const { t } = useI18N()
+
+    const poolURL = usePoolURL(pool)
+    const chainId = useChainId()
+    const [prize, setPrize] = useState('TBD')
```

Comment:
👀

Reply:
I did this to set a default value for the prize, is this not a good way?

Reply:
I thought this is a "TODO"